Update Language.ko.xml

initbar opened the pull request:
Linking #30

idrassi commented:
Can you please explain the reason for modifying existing translations that are inherited from TrueCrypt (e.i. IDM_VERIFY_RESCUE_DISK_ISO & TOKEN_DATA_OBJECT_LABEL)?

initbar commented:
Yup! - I'll provide explanations for my translations in the following:

#### 14eaa90
<table>
  <tr>
    <td width="33%"><img src="https://cloud.githubusercontent.com/assets/12630151/10873427/91e93cc6-80e2-11e5-90c0-c3ac45b2805d.png"></td>
    <td width="33%"><img src="https://cloud.githubusercontent.com/assets/12630151/10873428/91e9cf74-80e2-11e5-9dac-fed14bd8b2a0.png"></td>
    <td width="33%"><img src="https://cloud.githubusercontent.com/assets/12630151/10873426/91e7c1de-80e2-11e5-9d62-91623df4339d.png"></td>
  </tr>
  <tr>
    <td>In this commit, I've took a quick gander and focused on three patterns. The first pattern (above) was areas in which (1) short and medium-length terms|remarks could be translated without much hardship. One of the most important part was keeping the terms consistent with those used by mister Kim Young without fragmenting all over the place with similar synonyms.</td>
    <td>(2) Keeping the general structure by removing the space after colon.</td>
    <td>(3) trailing whitespaces.</td>
  </tr>
</table>
#### 3eb383d
<table>
  <tr>
    <td width="33%"><img src="https://cloud.githubusercontent.com/assets/12630151/10873579/d6c0b688-80e4-11e5-8fc9-cb681d6ba1f6.png"></td>
    <td width="33%"><img src="https://cloud.githubusercontent.com/assets/12630151/10873580/d6c19c06-80e4-11e5-983c-4ed24228dc94.png"></td>
    <td width="33%"><img src="https://cloud.githubusercontent.com/assets/12630151/10873581/d6c1d3ec-80e4-11e5-9305-1840809aa283.png"></td>
  </tr>
  <tr>
    <td>I've reverted my mistake <code>~</code> back to <code>&lt;-&gt;</code> after checking that other language had used <code>&lt;-&gt;</code> symbol.</td>
    <td>Translation of longer* sentences.</td>
    <td>Correcting <code>lang=""</code> attribute values.</td>
  </tr>
</table>
Now up to this point, mister Kim's translations weren't touched for the following final revision (commit):
#### 6691cc0
Initially, `IDM_VERIFY_RESCUE_DISK_ISO` and consequent `Favorites` translations were in English (except the last element):

![3_01](https://cloud.githubusercontent.com/assets/12630151/10873723/b1a983aa-80e6-11e5-8b54-582a0f67d6a7.png)

In commit `14eaa90` line 264, I've translated to `응급복구 디스크 ISO 완전성 검증` - which later I've realized that using `완전성`, which means "holistic, on the whole, integrity" would be redundant and aesthetically unappealing due to inconsistency (even with and without the extra 3 characters, users would still be able to correctly infer):

![3_02](https://cloud.githubusercontent.com/assets/12630151/10873774/8b639626-80e7-11e5-83d9-39c3cb353f08.png)

As for `TOKEN_DATA_OBJECT_LABEL`, mister Kim's translation is "technically" correct, as the literal translation is: "file name". However, in terms of tokens, I've revised to `토큰 레이블`, which reads "token label". I thought this would be less confusing.

In `LABEL_ITEM`, mister Kim's translation was `라벨` - which does translate to "label", but reads "lah-bell". I preferred if it was consistent with the terms used in other places, and changed to `레이블` (which literally reads "label"). In a minor note, in `IDT_FAVORITE_LABEL`, he uses `레이블` instead of `라벨`.

The `IDT_NUMBER_KEYFILES` to `IDT_KEYFILES_BASE_NAME`'s removal of `의` is because (1) the `의` pushes the following characters out of the windows and hide the them from view. (2) `의` just (roughly) means: "of". The overall translation is much more clean to remove this and directly say:

| from | to |
|:------:|:---:|
| `키파일 수:` | ~Keyfile numbers: |
| `키파일 사이즈 (Byte):` | ~Keyfile size (Byte): |
| `키파일 배이스 이름:` | ~Keyfile basename: |

So after this revision, it looks like:

![3_03](https://cloud.githubusercontent.com/assets/12630151/10873914/b82d5550-80e9-11e5-9092-a0ef24696747.png)
